ios/browser/api/ads/brave_ads.mm

```diff
@@ -25,6 +25,7 @@
 #include "base/time/time.h"
 #include "base/values.h"
 #import "brave/build/ios/mojom/cpp_transformations.h"
+#include "brave/components/brave_ads/core/mojom/brave_ads.mojom-forward.h"
```

Comment:
forward header is not needed because `brave_ads.mojom.h` below.

Reply:
Great catch, and were fixed in a follow-up, I will raise the PR today. Thanks